app/(api)/auth/user/route.ts

```diff
@@ -0,0 +1,9 @@
+import { GET } from './GET';
```

Comment:
I see you use capital letters for the file name, this is fine as long as you and Win agree on this. Ask him about it to confirm.